### ISSUE TITLE
Language folders not always removed after embedding

### ISSUE THREAD

**Issue by tdsrhh** [MarcStan/Resource.Embedder#14]
Some times language folder en-US is not removed after embedding resources.

The build window reports :

1>  Deleted resource files 'en-US, nb-NO' as they where embedded into the target.
1>  Deleted resource directories 'en-US, nb-NO' as they are empty.

But the folder en-US is still present in the bin folder, even if the folder itself is empty.

**Comment by MarcStan** [comment]
I believe this is a regression after #9 as I've seen this behaviour as well (unfortunately only on one machine and not reproducible enough to debug it).

Delete used to happen very late in the build but now it happens a lot sooner (directly after assemblies are generated).

So I suspect some concurrency issue where another task will create the directories if they don't exist, but I have no clue which or why.

I will have to investigate.

As a workaround you can revert to 1.2.6 for now and wait for the next release. (1.2.7/1.2.8. only introduced the PostBuild support like #9 describes).
